src/Libraries/Microsoft.Extensions.AI.Abstractions/Tools/HostedMcpServerTool.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
There are other tools that you've not moved into the Tools folder, right? e.g. HostedFileSearchTool? What about AIFunction?

Reply:
I would prefer keeping AIFuncion in the Functions folder, seems to me it deserves its own directory at this level, otherwise, we would have to have Tools/Functions.  Aside from that, I only see HostedFileSearchTool, which I prefer we move to Tools too.

 HostedFileSearchTool was merged after I opened this PR, and I must've missed them while rebasing.